src/debugging.md

```diff
@@ -0,0 +1,116 @@
+# Debugging your scripts locally
+
+For a complete example, see the [debugging example](https://github.com/SubstraFoundation/substra/tree/master/examples/debugging).
+
+- [Debugging your scripts locally](#debugging-your-scripts-locally)
+  - [Dependencies](#dependencies)
+  - [Run your code locally](#run-your-code-locally)
+    - [Use assets from a deployed Substra platform and local assets](#use-assets-from-a-deployed-substra-platform-and-local-assets)
+    - [Dependency between assets](#dependency-between-assets)
+    - [Debugging with logs and breakpoints](#debugging-with-logs-and-breakpoints)
+      - [Access the logs](#access-the-logs)
+      - [Debug with pdb](#debug-with-pdb)
+
+## Dependencies
+
+- Substra must be installed.
+
+- Docker is required since the training and testing tasks are run in Docker containers:
+  - Mac: [Docker Desktop](https://www.docker.com/products/docker-desktop)
+  - Ubuntu/Debian: `sudo apt install docker`
```

Comment:
It is possible to add the link to the official documentation here as it is  important to add the gpg key: https://docs.docker.com/engine/install/ubuntu/

Reply:
done!